lib/jsdom/living/node.js

```diff
@@ -69,6 +73,21 @@ module.exports.clone = function (node, document, cloneChildren) {
   return copy;
 };
 
+function copyAttributeList(sourceElement, destElement, document) {
+  // Needed by https://dom.spec.whatwg.org/#concept-node-clone
```

Comment:
This doesn’t seem to be a separate function according to the spec. Instead, it’s folded into the cloning algorithm directly. See https://dom.spec.whatwg.org/#concept-node-clone.

Additionally, the spec just uses “clone” to create the attributes these days. So no need to use document._createAttribute below.